apps/rewards/app/components/Panel/ViewReward/ViewReward.js

```diff
@@ -126,16 +129,16 @@ class ViewReward extends React.Component {
             <div>
               <FieldTitle>Period</FieldTitle>
               <div>
-                {Intl.DateTimeFormat().format(startDate)}
+                {format(startDate, 'd-m-yyyy')}
```

Comment:
I don't think it's wise to hard-code the format here or on line 134, since it varies by locale and this format will confuse any Americans without more context. 

I utilized the `Intl` object specifically because it is locale-sensitve and will show the date in a way users are expecting to see it.

Reply:
Good point. I wondered about it before, but went with design. But I guess we can as well start thinking about i18n now. Changed.